### PR TITLE
Create basic configuration and info files

### LINKED ISSUE #6: Add basic configuration and info files
* [x] `README.md`, stating your project title, styled with Markdown
* [x] `LICENSE` file. Find a template for it in plain text or Markdown format. You may use MIT license for simplicity
* [x] `.gitattributes`. Find some template for your repo on github
* [x] `.gitignore`. Find a typical template for your language (Python, Vim, Linux etc.) @ gitignore.io

You'll need to push a new branch (e.g. `features/3-initial-config`) with atomic commits and [create a PR](https://help.github.com/en/articles/creating-a-pull-request) **referencing this issue**.

### PR DESCRIPTION
**Pull request by RootenberG**
Closes #6

### CODE REVIEW

**Comment by webknjaz** [comment]
Extend the commit message with a longer description explaining what it does more precisely.

**Comment by RootenberG** [comment]
> Extend the commit message with a longer description explaining what it does more precisely.


But everyone can see the list of changes using the ```git log -p``` command, I think it is not difficult to understand what is happening.

**Comment by webknjaz** [comment]
People rarely look at patch contents before checking the commits list. The history should make sense to attract the reader to certain commit so that they'd look into it closer.

**Comment by webknjaz** [comment]
> to simplify the work on the project

It sounds like an advertisement "hey, apply this cool commit", but you're not writing a novel.

This is too abstract. Be more pragmatic. Explain how this commit is useful on the high level.

It should add value. For obvious things like these configs controlling Git behavior you don't need to explain what they do — the reader already uses Git and they either already know what those files are or know where to find such info.

I'd do something like this:
```
Create initial git config files

Specify:
* .gitignore
* .gitatributes
```

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+*.pkl 		binary
+*.pyc 		binary
+*.pyd		binary
+*.pyo 		binary
+
+# Note: .db, .p, and .pkl files are associated
+# with the python modules ``pickle``, ``dbm.*``,
+# ``shelve``, ``marshal``, ``anydbm``, & ``bsddb``
+# (among others).
+
```
Comment (webknjaz) [inline .gitattributes]: Could you please drop trailing empty lines in the end and at the beginning of this file?

**Comment by webknjaz** [comment]
Also, let's do #14 before merging this

File: .gitattributes (hunk 1 above)
Comment (RootenberG) [reply .gitattributes]: No problem

**Comment by webknjaz** [comment]
Now squash related changes and let's wait for #14

**Comment by webknjaz** [comment]
Let's merge #12 first

**Review by webknjaz — CHANGES_REQUESTED** [review]
I just realized that the file names of Git-related configs you added are wrong. Fix that.

**Review by webknjaz — DISMISSED** [review]
Ah, sorry. It's a bug on GitHub :)

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Basic .gitattributes for a python repo.
```
Comment (webknjaz) [inline .gitattributes:1]: I'd also recommend setting a default for text file EOLs to LF

Comment (webknjaz) [reply .gitattributes:1]: smth like `* text=auto eol=lf` (need to check the syntax tho)

Comment (serhii73) [reply .gitattributes:1]: @RootenberG you can see examples [here](https://github.com/alexkaratarakis/gitattributes/search?q=eol&unscoped_q=eol)

**Review by webknjaz — APPROVED** [review]

**Pull request merged**